### PR TITLE
quick quick fix

### PR DESCRIPTION
**Pull request by glanau**

### CODE REVIEW

**Comment by ElevenWilson** [comment]
51 words might be misspelled, please check them: AngularJS, AuthGuard, AuthHttp, authHttpServiceFactory, canActivate, CLI, creer, css, d'Angular, dauthentification, DoctrineFixturesBundle, finaliser, FOSRestBundle, gary, glanau, html, Http, JMSSerializer, JMSSerializerBundle, Json, jwt, JWT, JWTs, l'API, l'authentication, Lexik, LexikJwtAuthenticationBundle, LexikJWTAuthenticationBundle, l'id, l'installeur, LoadPostData, localStorage, login, logout, l'url, NelmioCorsBundle, ng, ngModel, npm, png, PostController, ReactiveFormsModule, s'authentifier, symfony, Symfony, systeme, tutoriel, url, username, validateurs, yml

File: _posts/2017-03-03-angular2-symfony3-comment-creer-rapidement-systeme-dauthentification.md [hunk 1]
```diff
@@ -44,8 +44,7 @@ Sommaire
 
 ![](/assets/2017-03-03-angular2-symfony3-comment-creer-rapidement-systeme-dauthentification/sf-blog-2.png)
 
-----------------
----------------------------------------------------------------------------------------------------------------------------------
+---------------------------------------------------------------------------------------------------------------------------------------------
```
Comment (VEBERArnaud) [inline _posts/2017-03-03-angular2-symfony3-comment-creer-rapidement-systeme-dauthentification.md]: `---` (trois tirets) suffisent 😄 

exemple:

---

Comment (glanau) [reply _posts/2017-03-03-angular2-symfony3-comment-creer-rapidement-systeme-dauthentification.md]: yes ;)

**Comment by ElevenWilson** [comment]
51 words might be misspelled, please check them: AngularJS, AuthGuard, AuthHttp, authHttpServiceFactory, canActivate, CLI, creer, css, d'Angular, dauthentification, DoctrineFixturesBundle, finaliser, FOSRestBundle, gary, glanau, html, Http, JMSSerializer, JMSSerializerBundle, Json, jwt, JWT, JWTs, l'API, l'authentication, Lexik, LexikJwtAuthenticationBundle, LexikJWTAuthenticationBundle, l'id, l'installeur, LoadPostData, localStorage, login, logout, l'url, NelmioCorsBundle, ng, ngModel, npm, png, PostController, ReactiveFormsModule, s'authentifier, symfony, Symfony, systeme, tutoriel, url, username, validateurs, yml

**Pull request merged**